bin/lib/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+services:
+  cloud:
+    image: civiform/civiform-cloud-deployment:${CIVIFORM_CLOUD_DEPLOYMENT_VERSION}
+    environment:
+      - AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY
+      - AWS_SESSION_TOKEN
+      - AWS_REGION
+      - AZURE_CLIENT_ID
+      - AZURE_STORAGE_ACCOUNT_NAME
+      - AZURE_LOCAL_CONNECTION_STRING
+      - AZURE_STORAGE_ACCOUNT_CONTAINER
+      - CONFIG_FILE
+      - IMAGE_TAG
+      - COMMAND
+    volumes:
+      # Mounting civiform config to the image in read only mode.
+      - ${PWD}/${CONFIG_FILE}:/${CONFIG_FILE}:ro
+    command: cloud/shared/bin/run.py --tag=${IMAGE_TAG} --command=${COMMAND} --config=${CONFIG}
```

Comment:
We use CONFIG_FILE and CONFIG. What's the difference between them?

Reply:
there's no reason. Renamed.